__tests__/UglifyJsPlugin.test.js

```diff
@@ -0,0 +1,824 @@
+"use strict";
```

Comment:
Sure, but can you clarify? Do u mean put snapshot tests in diff file? Or have the snapshots themselves be separated?